### ISSUE TITLE
phosphatase interactors annotated as negative regulators (reviewing unknowns)

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#3711]
This paper identifies interactors of PP1
http://amigo.geneontology.org/amigo/search/annotation?q=PMID:%2019389623
 
47 of these are annotated to

negative regulation of phosphatase activity

although they are only interactions.

I don't know the best way to report this, because it is too time consuming to raise a dispute for all 47 in protein2GO. I will dispute one and tag this ticket.

**Comment by sylvainpoux** [comment]
Hi Val,

you are right, I deleted these annotations. By the way, I regret to say that GO systems for reporting errors are not that efficient. There should be a system where a bunch of editors (Pascale and someone else, for example) have writing permission. It would really smooth things 

I have such right for UniProt in Protein2GO (a great and efficient tool) and it really simplifies things. On the down side, I am assigned to most UniProt requests...

Thanks

Sylvain

**Comment by ValWood** [comment]
But this isn't really a reporting problem, it's a  general annotation maintenance issue. I think @pgaudet  already fixes the errors from all of the non-maintained sources. But shouldn't each database fix its own errors? This way they can feedback to curators and update curation rules/training materials. It would be a lot of work for the people reporting, more for Pascale to go into 47 entries in Protein2GO and fix them.

I reported this problem via the helpdesk, and  protein2GO dispute, but I  only tagged you to assign to the appropriate person because I saw it was from a 'retired curator'.
